api/java/manipulating-tables/table_list.md

```diff
@@ -11,16 +11,19 @@ related_commands:
 # Command syntax #
 
 {% apibody %}
-db.tableList() &rarr; array
+db.tableList() &rarr; TableList
 {% endapibody %}
 
 # Description #
 
 List all table names in a database. The result is a list of strings.
 
-__Example:__ List all tables of the 'test' database.
+__Example:__ List all tables of the 'test' (`DEFAULT_DB_NAME`) database.
```

Comment:
@clovergaze why do we need to add the 'DEAULT_DB_NAME? The examples should be copy-paste snippets.